lib/tasks/fetch_and_sync_all_pull_requests_reputation.rake

```diff
@@ -0,0 +1,4 @@
+desc 'Fetch and sync all pull requests reputation'
+task fetch_and_sync_all_pull_requests_reputation_job: :environment do
+  FetchAndSyncAllPullRequestsReputationJob.perform_now
```

Comment:
I've also removed the organization members sync rake task, as that is also done here via `FetchAndSyncAllPullRequestsReputationJob`